codec-http/src/main/java/io/netty/handler/codec/http/cookie/ServerCookieDecoder.java

```diff
@@ -75,7 +75,7 @@ private ServerCookieDecoder(boolean strict) {
         boolean rfc2965Style = false;
         if (header.regionMatches(true, 0, RFC2965_VERSION, 0, RFC2965_VERSION.length())) {
             // RFC 2965 style cookie, move to after version value
-            i = header.indexOf(';') + 1;
+            i = getDelimiterPos(header, 0) + 1;
```

Comment:
Also, if the delimiter is not found, the decoder will return the `$Version`->'1'? Is it's fine?